### DISCUSSION TITLE
Welcome to open-source-mac-os-apps Discussions!

### DISCUSSION THREAD

**Discussion by serhii-londar** [serhii-londar/open-source-mac-os-apps#627] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Williamehanksjr** [comment]
Hi everyone. I am a retiired software engineer looking to get back in as a hobbiest. I developed Chemistry software on a Mac in the early 90's in c++. Any suggestions.

**Comment by A-Alanbri** [comment]
> 
> 
> ## 👋 Welcome!
> 
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>   * Ask questions you’re wondering about.
> 
>   * Share ideas.
> 
>   * Engage with other community members.
> 
>   * Welcome others and are open-minded. Remember that this is a community we build together 💪.
> 
> 
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 
> 
> 
>

**Comment by bysiber** [comment]
Hey everyone! I wanted to share a project that I think fits perfectly in this collection.

**[ClearDisk](https://github.com/bysiber/cleardisk)** -- a free, open-source macOS menu bar app (Swift/SwiftUI) that helps developers reclaim disk space by cleaning developer caches like Xcode DerivedData, node_modules, Docker images, CocoaPods, Homebrew, pip, Cargo, Go modules, and more.

It runs natively on macOS 14+ with a clean SwiftUI interface -- no Electron, no subscriptions. You just click the menu bar icon, see what is eating your disk, and clean it.

I've already submitted a [PR to add ClearDisk to the list](https://github.com/serhii-londar/open-source-mac-os-apps/pull/1202), but wanted to share it in Discussions too since the Welcome post mentioned sharing ideas and projects.

You can install it via Homebrew:
```
brew tap bysiber/cleardisk && brew install --cask cleardisk
```

Or download the DMG from the [Releases page](https://github.com/bysiber/cleardisk/releases).

Would love feedback from the community!
